File: MODUL.4/PRAK404_2310817120010_NurHikmah.py
while True:
    print("\nPilih program")
    print("1. Penjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Exit")
    pilihan = int(input("Masukkan Pilihan : "))

    if 1 <= pilihan <= 4:
        nilai1 = float(input("Masukkan nilai pertama : "))
        nilai2 = float(input("Masukkan nilai kedua : "))

        if pilihan == 1:
            hasil = nilai1 + nilai2
            print(f"Hasil penjumlahan antara {nilai1:.2f} dengan {nilai2:.2f} adalah {hasil:.2f}")
        elif pilihan == 2:
            hasil = nilai1 - nilai2
            print(f"Hasil pengurangan antara {nilai1:.2f} dengan {nilai2:.2f} adalah {hasil:.2f}")
        elif pilihan == 3:
            hasil = nilai1 * nilai2
            print(f"Hasil perkalian antara {nilai1:.2f} dengan {nilai2:.2f} adalah {hasil:.2f}")
        elif pilihan == 4:
            hasil = nilai1 / nilai2
            print(f"Hasil pembagian antara {nilai1:.2f} dengan {nilai2:.2f} adalah {hasil:.2f}")
    elif pilihan == 5:
        print("Terimakasih, telah menggunakan kalkulator NURHIKMAH")
        break
    else:
        print("Input anda salah, silahkan coba lagi")

File: MODUL.4/PRAK404_2310817120010_NurHikmah.c
#include <stdio.h>

int main()
{
    int pilihan;
    float nilai1, nilai2, hasil;

    do
    {
        printf("\nPilih program\n");
        printf("1. Penjumlahan\n");
        printf("2. Pengurangan\n");
        printf("3. Perkalian\n");
        printf("4. Pembagian\n");
        printf("5. Exit\n");
        printf("Masukkan Pilihan : ");
        scanf("%d", &pilihan);

        if (pilihan >= 1 && pilihan <= 4)
        {
            printf("Masukkan nilai pertama : ");
            scanf("%f", &nilai1);
            printf("Masukkan nilai kedua : ");
            scanf("%f", &nilai2);

            switch (pilihan)
            {
            case 1:
                hasil = nilai1 + nilai2;
                printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f\n", nilai1, nilai2, hasil);
                break;
            case 2:
                hasil = nilai1 - nilai2;
                printf("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f\n", nilai1, nilai2, hasil);
                break;
            case 3:
                hasil = nilai1 * nilai2;
                printf("Hasil perkalian antara %.2f dengan %.2f adalah %.2f\n", nilai1, nilai2, hasil);
                break;
            case 4:
                hasil = nilai1 / nilai2;
                printf("Hasil pembagian antara %.2f dengan %.2f adalah %.2f\n", nilai1, nilai2, hasil);
                break;
            }
        }
        else if (pilihan == 5)
        {
            printf("Terimakasih, telah menggunakan kalkulator NURHIKMAH\n");
            break;
        }
        else
        {
            printf("Input anda salah, silahkan coba lagi\n");
        }

    } while (1);

    return 0;
}